virtualizarr/tests/test_xarray.py

```diff
@@ -271,6 +272,19 @@ def test_combine_by_coords(self, netcdf4_files):
         assert combined_vds.xindexes["time"].to_pandas_index().is_monotonic_increasing
 
 
+
+
+
+
```

Comment:
There are lots of linting things in this PR like this that our tools should be correcting, see #96